design-system-docs/frontend/styles/_header.scss

```diff
@@ -0,0 +1,47 @@
+// ============================================================================
+// Docs header overrides
+// ============================================================================
+
+.cads-header {
+  border-bottom: $cads-border-width-small solid $cads-palette__white;
+
+  // Allow logo row to use auto-width
+  // @TODO: Currently uses grid styles and must always be 2-cols, which seems wrong?
+  .cads-header__logo-row {
+    width: auto;
+    max-width: none;
+  }
```

Comment:
Noticed this is a problem on products, where the logos actually overflow technically. The header might make sense using more forgiving grid styles here.